homework/style.css

```diff
@@ -1,3 +1,86 @@
 .alert-error {
```

Comment:
You have still some work to do to make your app responsive:

![image](https://user-images.githubusercontent.com/2788771/59921955-a2f74a80-942f-11e9-92ac-d422e1b38f16.png)

Reply:
Thanks! I will fix it soon!